py3status/modules/yandexdisk_status.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+"""
+Display Yandex.Disk status.
+
+Configuration parameters:
+    cache_timeout: how often we refresh this module in seconds (10s default)
+    format: prefix text for the Yandex.Disk status
+
+Valid status values include:
+    - Yandex.Disk isn't running!
+    - Error: daemon not started
+    - ...
```

Comment:
@tobes 
I'ts a bit strange but when I run `py3status docstring update yandexdisk_status` it says me `py3status: Setup error (IOError) docstrings.py line 26`.

At this line: `readme_file = os.path.join(modules_directory(), 'README.md')`
I've printed the `readme_file` value: `/usr/local/lib/python2.7/dist-packages/py3status-2.9-py2.7.egg/py3status/modules/README.md`. But there is no file with this name. I'm not sure but maybe it's a bug.